pyop2/types/glob.py

```diff
@@ -203,6 +203,14 @@ def inner(self, other):
         assert issubclass(type(other), type(self))
         return np.dot(self.data_ro, np.conj(other.data_ro))
 
+    def axpy(self, alpha, other):
+        """Compute the operation :math:`y = \\alpha x + y`.
+        """
```

Comment:
docstring needs the parameter list (and see comments above about argument naming)